packages/frontend-api/src/Model/Resolver/Category/CategoriesResolver.php

```diff
@@ -21,14 +26,21 @@ class CategoriesResolver implements ResolverInterface, AliasedInterface
      */
     protected $domain;
 
+    /**
+     * @var \Shopsys\FrontendApiBundle\Model\Category\CategoryFacade
+     */
+    protected FrontendApiCategoryFacade $frontendApiCategoryFacade;
+
     /**
      * @param \Shopsys\FrameworkBundle\Model\Category\CategoryFacade $categoryFacade
      * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+     * @param \Shopsys\FrontendApiBundle\Model\Category\CategoryFacade $frontendApiCategoryFacade
      */
-    public function __construct(CategoryFacade $categoryFacade, Domain $domain)
+    public function __construct(CategoryFacade $categoryFacade, Domain $domain, FrontendApiCategoryFacade $frontendApiCategoryFacade)
```

Comment:
I'm afraid this is a BC break (CategoriesResolver is in the already released version).

I  think a new resolver for search could solve it nicely without the need of setter injection

Reply:
f4d5adb729dce11f68d60db2146c0046a728915d

Reply:
6785a0fa97a94376e407474bf7f4d897e0da3a68